<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeChildView</class>
 <widget class="QWidget" name="TreeChildView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="childHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblLocationImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_location.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblFavButton">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>7</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_favourite_outline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblLockImage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>7</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_radio_unselected.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblChildName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>8</y>
      <width>80</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>City Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
